### DISCUSSION TITLE
problems with official US server

### DISCUSSION THREAD

**Discussion by johnrechard85** [BearWare/TeamTalk5#1793] [General]
Hello everyone! I am writing this post to bring to your attention an individual who has been causing trouble in Team Talk community. Her name is Shamika Henry, and she is running a scam that has affected many individuals on the US server. Shamika has been asking people for money with the promise of promoting them to moderators on the server.

She has done this to at least five or six individuals, and they have all reported that she did not honor her promise. To make matters worse, she has recently scammed one person out of their money and then banned their username from the server for no valid reason. This is highly unfair, and we are calling on the server's main administrator, Bjorn, to take action against Shamika Henry as soon as possible.

Several individuals, including myself, have reached out to the administrator via email, but we have yet to receive a response. We are hoping that by bringing this issue to the this place, we will get the necessary attention required to rectify this situation.

It is important that Shamika is replaced with someone who is eligible to moderate the server and takes her out as soon as possible. We cannot continue to allow her to cause distress to members of the community and engage in such fraudulent behavior.

We appreciate your attention to this matter and are hoping for a prompt resolution to this issue.

Shamika has been engaging in fraudulent activities and has been causing a great deal of distress to multiple members on the server. 

Shamika has been asking for money from several members of the server, promising to promote them as moderators to the server's main administrator. This is not only unethical, but it is also unfair to others who are looking to earn a promotion through hard work and dedication. To make matters worse, she scammed one of the members who paid her the money, and then banned that person's username for no reason.

Despite several attempts made by me and a few others, we have not received any response from the administrator regarding this matter. It is my sincere belief that immediate action needs to be taken against Shamika. She should be removed from her position as a moderator and replaced with someone who is eligible and capable of fulfilling the role without causing any further problems.

As members of this server, we all have a right to feel safe and supported by our fellow members, but Shamika's actions are making it difficult for us to feel that way. It is our hope that the administrator takes prompt and decisive action to resolve this issue, punish Shamika for her fraudulent behavior, and ensure that fair server policies are upheld.
Due to the official status of this server, it is our belief that it is Bjorn's responsibility to rectify this problem. Additionally, we want that we bring this matter to his attention immediately in order to ensure a prompt resolution.
Also, we are dismayed by what she wrote about Trump in the US server, as we strongly believe that Trump is a widely-liked personality in the United States and she should refrain from introducing political matters into a publicly-accessible server.
Thank you for your time and consideration in this matter.

Sincerely,

**Comment by ahmednaalik51** [comment]
I used official server in past, and also discover problems with this person.
She to kick and ban the users even some times random for no reason.
Please do some thing.

**Reply by johnrechard85** [reply]
We need bjorn to take action against this person as soon as possible.

**Comment by bear101** [comment]
I've talked to Shamika and sorted out the issue regarding new moderators. The 'Troy' user account has been banned due to abuse. Unfortunately the user who originally signed up for this moderator account ended up abusing it.

**Reply by computertech1012** [reply]
Hi,
@bear101 I can confirm that this is a problem on the official US and UK servers. It is not this other user you speak of, but another user Shamika who is abusing hoer moderation powers and offering money to people, including myself, to get power on the servers.
I don't understand why you only listen to her, are you wanting your servers to be unsafe due to your actions of not listening to her? I have sent you several emails, and you seam to not care about your servers, which is sad, because you are the maintainter, and if Shamika answers, you just instantly believe her and don't even allow the users to explain. That's a very sad way to run a company, I apologize but I need to speak my mind in this matter.
She has abused her powers, and gotten some other moderators who were good, removed for no reason. She has also told people that she will give them admin rights for money.
If this is going to continue, I would recommend that people find other programs to use, because it seams like the dev of TeamTalk does not care about his users, and wants people to feel unsafe on his servers.
I do not care if I get blocked from GitHub but this has been an issue for a long time and Bjorn does nothing to resolve it, he just continues to let her do whatever she wants, as if she's the queen of TeamTalk.
So if you want to ban me from the repo, that's fine, it only shows how twisted up this situation is.
Thanks for reading!

**Reply by johnrechard85** [reply]
Hi there,
Recently, I've heard many people claim that Shamika has been charging anywhere from $300 to $800 to make individuals moderators in US servers. Although I'm not familiar with Troy myself, several others have reported similar experiences. 
Additionally, I'm curious why Shamika is kicking and banning those who simply comment in discussions and also, why is she posting political topics in channels. It's important to remain impartial and not take sides. 
Thanks for your time."

**Comment by ghost** [comment]
i am agreed with 
creater.

**Comment by ahmednaalik51** [comment]
I don't to use the ofycial servers in teamtalk because of this person who was there for almost 2 years, banning and kicking the user from the server.
So I agree with the people who post in this topic.
It seams that BearWare101 didn't understand what is happened.
I don't know about the scams, but I know she abuse her power.
@computertech1012 your comment was very nasty but I understand what you say.
But I think the reason bearware101 does not listen is because he also know shameka for long time but also he does not care about it but he needs to.
Even we don't feel safe on the encrypted server because we do not know if she will come to move into our channels and read channel pass words.
So that's my fact on this matter.

**Comment by ChrisDuffley** [comment]
This is why I do not and will not go on the public servers. I have been and will continue to host my own. That is all I will (and can) say.

**Reply by computertech1012** [reply]
Hi,
@ChrisDuffley I agree with you 100%. It's sad that Bjorn won't even allow people to tell their sides of the story, because Princiss Shamika gets involved, tells her side, and then Bjorn instantly believes her with no more discussions.
If he would at least want to be fair, he'd allow people to show proof and other things. I also know about her scamming people for money, and I kknow of the people who she has scammed for the money as well.
As for the Trump Topic, I agree, very disrespectful regardless of if anyone agrees about Trump or not, there should be no politics in the TeamTalk servers, and it doesn't matter if she's joking or not.
You know what's really sad? Bjorn is in Denmark, and Shamika is in Trinidad. Neither one of them are from the US, yet they run a US server, how is that even ethical? And how is it okay that Shamika can post topics about Trump when she's not even in the US?
It's very very very sad that Bjorn does nothing about this, just bows down to her as if she's his queen, and she can do whatever she wants on the servers. If she asked him to destroy the server, I bet he'd do it. He does whatever she wants, and anyone that's friends with her he listens to as well. Very sad.
Thanks for reading!

**Comment by computertech1012** [comment]
Hi,
I do apologize for my rudeness in this topic. However, it's the truth. Someone needs to come out in the open and say it. He does not even care. If he did, he'd be willing to go into a call with some of the people who are saying these things and get both sides of the story. However, he only takes Shamika's side. I bet you all that he won't even reply to this topic. And yeah, I get it my comments were super rude, and I apologize. But I got an email about this topic, and I needed to speak my mind!
And he will probably say that we should just go to other servers, well how is that fair to people who are in the US who don't like the disrespect and other stuff that is happening on the US server by someone who is not even in the US.
Another thing, yeah I get it people spam, but yet she allows people with stupid nick names to be up on the server. And I don't know who he is, but if Bjorn took the time to read his logs, he would see that the only person who is abusing their powers is Shamika. And it's sad because he does not do anything about it, yet they are official servers, how can they be official servers when you have an administrator or mod or whatever you cal it, abusing her powers on the server. And I am not the only one who thinks this way, as you can see, this topic is a little active right now.
Like I said, I apologize for my rudeness. She will get someone to have a mod on the server, and then she will say that they are abusing their powers simply because she doesn't get what she wants and then wants them removed. Just like the musicbot username on the server, she gave it to people for money, she also gave it to an arabic person instead of using it to host a MusicBot herself, and then gave it to another person because they gave her money. Not to mension other things, like giving people an op in the root and other things, that Bjorn probably won't even care about.
And there's proof of it as well, I've sceen it, in fact, I am actually a victom of it.
But it's whatever, just use other servers, or use the encrypted server, because bearware.dk is known for now having servers that have people that abuse their powers. And the trump thing still bothers me, her topic in the Music Room UK about Trump is not cool, and it's not even funny.
Thanks for reading!

**Reply by johnrechard85** [reply]
I certainly agree with you.

**Comment by Russell-Bradler** [comment]
hi there

**Reply by johnrechard85** [reply]
Oh, I think learning how to spell words is not a hard task. So you've come to the conclusion that it's pointless to discuss the issue with us, have you? I'm sure everyone here is just thrilled to hear that. After all, what could possibly be the point of talking to people who don't understand you? Especially when you just want to get your own point across, am I right?
Well, I must say that your recommendation to other people not to use the official servers is certainly a bold move. Because apparently, you've decided that the situation is just too dangerous for anyone else to handle. You know, it takes a real hero to make such a noble sacrifice for the well-being of others. Bravo!
We certainly appreciate your contribution to our little discussion here, and we're all so grateful for your valuable insight. But, if I may offer a suggestion, perhaps instead of giving up on communication altogether, you could try to express your views in a way that others can understand? Just a thought. Anyway, thanks for reading, and good luck with your server boycott.

**Comment by Russell-Bradler** [comment]
i am dissagreed with the creater  100 %m because it's really rediculess to actually grab a life storry 😂, and I think you should stop that shit.

**Comment by ahmednaalik51** [comment]
I just will stay off of those servers in till bearware101 will be willing to fix.
I think he is also busy so give it time.
Please though bearware101 at least look into this problem.

**Reply by johnrechard85** [reply]
Actually I did the same thing.

**Comment by Russell-Bradler** [comment]
well, TBH I took a brake from official servers and the tottal teamtalk as well, because that's a place of drama.

**Reply by ghost** [reply]
Hay, why you are even writing this post if you are not in teamtalk comunity?
and yeah, people are all here are disscussing about the teamtalk problem.

**Reply by johnrechard85** [reply]
Well, well, well, it seems like we have a real communication breakdown here. Actually, I left the official servers after I've seen these things. I mean, I understand that you think it's pointless to talk about the problem because you assume that I don't understand your point of view. But come on now, do you really think that I'm that dumb that I just want to blabber on and not listen to what you have to say? 
Anyway, I appreciate you reaching out and sharing your concerns with me. It's always productive to try and solve problems through discussion, even if we don't see eye to eye at first. 
Now, if you're recommending that other people avoid using the official servers, I think that's a bit extreme. I mean, what's the harm in letting people decide for themselves whether or not they want to use them? But, of course, I understand that you're frustrated with the current situation and you're entitled to express your opinion.

So, in conclusion, let's try to keep an open mind and communicate with each other respectfully, even if we don't always agree. Thanks for taking the time to talk to me.

**Reply by computertech1012** [reply]
Hi,
@johnrechard85 I agree with you on this. However, it seams that @bear101 only listens to her, and doesn't even give other people a chance to speak and understand the situation. Also, there are people who are her friends in which she pays money to in order to get what she wants.
On top of all that, a well nknown spammer in the blind community, Dr. Ramon, is an operator in the Music Room, and also is her friend. So if she wants him to say something he'll do it.
Just saying, this is so sad that he doesn't do anything, all he cares about is her, and only listens to her. And she takes full advantage of that and tells lies to make up for her actions on the server.
How about just reinstating the entire server, and not making any mods, to keep things fair, instead of having someone who abuses their powers, and having an owner that doesn't look at logs, instead just listens to her whenever she reports something, instantly listens to her with no discussion about it.
Very sad.
Thanks for reading!

**Reply by ghost** [reply]
AGREED

**Comment by Ramon-Salazar** [comment]
Well, what you wrote @computertech1012  isn't true, I'm not a spammer, and I have been to the server for a very long time.

You're just mad because you are being banned for many reasons, and spam is one of the main reason and you already know it.

Almost everyone know what you did earlier, oh even today few hours ago, and you keep changing your IP to get into the server.

Also, you and @johnrechard85 lied regarding Shamika, she never scammed everyone, and when there is an increase of spamming activities she premote more operators to the main channel or to the music box in order to mitigate this problem and to reduce this activity a lot.

**Reply by computertech1012** [reply]
Hi,
Actually you are the spammer @Ramon-Salazar but I will be ignoring your future replies because I have plenty of recordings of you spamming the server. However, your comments are off topic so I will say no more.
Thanks for reading!

**Comment by Ramon-Salazar** [comment]
That isn't true @computertech1012, and even if you have recordings that was in the past, also you wrote already and accused me of spamming which in fact not  true.

Even if you ignored this post the facts will be still there and it'll be shown at any times regarding you.

**Comment by Ramon-Salazar** [comment]
BTW, Absolutely true @Bear101, noticed that so called troy abused the power.

It appeared the same guy which is Cookie AKA Cameron beaver
@computertech1012, he ran behind many accounts, no wonder why he's so
rude, he's even lying regarding Shamika, as I know Shamika never
scammed everyone, and she's moderately ban spammers like him and
anyone who supports him, and do to increasing of spam activities she's
promoting more operators, but not for money as he claimed.

It appeared that the author @johnrechard85  is one of his supporters
or probably him running behind this account, and commenting to himself
to get more attention, seems most if not all  of the comments are by
him.

 He claimed that I'm spammer which in fact not true at all, despite
that he claimed to have recordings, probably these recordings are too
old and doesn't even apply to the present time or even near present
time.

**Comment by muamel1** [comment]
@computertech1012 , github is not a place for your drama cookie, You need to stop spreading rumors about other people just because you don't like them
Users may blame shamika for not banning you when you spam their channels, I understand why she is not doing so. The problem is in TeamTalk itself, not the administrators.
Even if she bans you, you will use a vpn to change your IP address and spam again
This is something that requires @bear101  attention, not what you wrote above

**Reply by ahmednaalik51** [reply]
It does not matter at this point who spammed what.
What matters is that Shamika abuse her powers in the teamtalk.
I think every one should have chance to tell their sides.
So everyone can tell the side.
I only know Shamika abuse her power in the teamtalk but I don't know about money.
I don't use the official server for only that reason.
So my idea for everyone wants to be to tell the side important.
That way it can be fairly resolved.
Defending your self in the chat about teamtalk does not solve anything.

**Comment by johnrechard85** [comment]
Hello all,
I'd like to redirect the conversation away from the spamming accusations and focus on a more pressing issue. Specifically, the fact that Shamika has abused several individuals on this platform. It's not uncommon for there to be drama and spam in public servers, but what concerns me is why Shamika is offering money to people in exchange for moderator status, and why she is posting political content about Donald Trump in channels that are unrelated to politics.
It's important that Shamika takes responsibility for her actions, and that the main administrator, Bare101, also addresses the situation. Let's shift the conversation from petty arguments to holding individuals accountable for their behavior. Thank you."
